pkgs/desktops/gnome-3/extensions/noannoyance/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv
+, lib
+, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "noannoyance-unstable";
+  version = "2021-01-17";
+
+  src = fetchFromGitHub {
+    owner = "BjoernDaase";
+    repo = "noannoyance";
+    rev = "f6e76916336aee2f7c4141796f3c40c870d2b347";
+    sha256 = "1iy3nif8rjjcwf83fg9ds93fi7vmhliynmlwqnx036s3msmxvgs3";
+  };
+
+  uuid = "noannoyance@daase.net";
+
+  dontBuild = true;
+
+  installPhase = ''
+    mkdir -p $out/share/gnome-shell/extensions/${uuid}
+    cp metadata.json extension.js $out/share/gnome-shell/extensions/${uuid}
+  '';
+
+  meta = with lib; {
+    description = "Removes the 'Window is ready' notification and puts the window into focus";
+    homepage = "https://github.com/BjoernDaase/noannoyance";
+    license = licenses.gpl2;
```

Comment:
gpl2Plus or gpl2Only?

Reply:
It does not specify "or later" anywhere, so I assume its gpl2Only, though I'm not an expert on licensing matters.